### DISCUSSION TITLE
小鹤双拼和音形

### DISCUSSION THREAD

**Discussion by kiyurei** [LufsX/rime#7] [Q&A]
想问一下，单纯的小鹤双拼和小鹤音形之间的配置文件的差别是什么，看了好久没看懂。两者如何转化呢？求解答

**Comment by LufsX** [comment]
应该是码表差异，小鹤音形是双拼 + 两位形的固定码，双拼部分都是一样的，只有形部分是新增的

我不用形我也不太清楚哈 🤣

**Comment by kiyurei** [comment]
我有试过其他的配置文件，因为想直接打英文，但是那种音形的，想改但不知道怎么弄，但还是谢谢回答

**Reply by LufsX** [reply]
音形不是四码上屏么，连打英文的话就直接上屏了啊

**Reply by kiyurei** [reply]
我的说法有问题。我用你的配置的话，英文输入起来有问题。我又有找其他的配置，但又是音形的。所以我想自己改，是这个意思。还有，我觉得你做的这个皮肤真好看，我现在就在用这个打字

**Reply by LufsX** [reply]
具体问题是啥？

迁移配置的话可以按着我博文改，大部份改的内容都写出来惹～
